### PR TITLE
Update docs of Slack integration

### PR DESCRIPTION
**Pull request by akito19**
We introduced the notification mode to control notification threshold.
Screen Shot 2019-03-12 at 15 58 03

### CODE REVIEW

**Comment by akito19** [comment]
@ivanha09 Would you review this PR?

File: docs/getting-started/repository-settings.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ This section allows you to enable the analysis tool you would like to use. Tools
 
 In this section, you can enable Slack notifications from Sider. Enter `team#channel` in the Name field and enter your Slack webhook url in the `URL` field.
 
+In addition, if you wish not to receive notifications for no issue analyses, choose `Errors Only`. Then, Sider will notify analysis results to your Slack channel only when it detects issues.
```
Comment (ivanha09) [inline docs/getting-started/repository-settings.md]: In addition, if you wish not to receive notifications for analyses with no issues, choose `Errors Only`. Then, Sider will notify analysis results to your Slack channel only when it detects issues.

Comment (ivanha09) [reply docs/getting-started/repository-settings.md]: I think the above sentence might read better if what you are trying to say is that Sider always sends notifications when an analysis is complete even when there are no issues. So to stop that choose 'errors only'.

Comment (akito19) [reply docs/getting-started/repository-settings.md]: Exactly. I'd like to inform that Sider always sends notifications to a Slack channel unless setting `Errors Only`. 
I'm going to fix that. Thank you!

**Comment by akito19** [comment]
I'm merging on completing building.

**Pull request merged**